Updated to @angular/material 2.0.0-beta.6 and @angular/* 4.1.3.

SamuelMarks opened the pull request:
Your repository was linked from Google's as the featured example. Unfortunately you have lapsed behind their codebase.

I've brought you back up to speed, specifically I've:

 0. Generated a new scaffold from `@angular/cli` 1.0.3;
 1. Implemented `@angular/router` routing with lazy loading;
 2. Followed the official `@angular/material` [install guide](https://github.com/angular/material2/blob/master/guides/getting-started.md);
 3. Used [step 3](https://github.com/angular/material2/blob/master/guides/getting-started.md#step-3-import-the-component-modules) suggestion to create one module for all `@angular/material` module imports;
  4. Migrated the relevant code from your example

Enjoy

SamuelMarks committed:
Updated to @angular/material 2.0.0-beta.6 and @angular/* 4.1.3.

m67hoff commented:
Hy Samuel,

i've seen your updates / PR  but I'm not so happy with them.  What I like is that you invested the effort to update it t the latest material / cli level.

however this is a beginners material 2 sample so why add ed routing ?  (this makes it more complicated to understand.) 
also: change the src dir to m2-app ?  (same reason)
you also included the now unused app.component default files  
and missed some material-modules in the import-module.  (sidenav / toolbar)

(anyway it seems this sample is not maintained anymore (if looking to the issues)   

Greetings from germany - michael

SamuelMarks committed:
Removed routing; removed extra module; added missing CSS & scripts; imported `MdToolbarModule`

SamuelMarks commented:
@m67hoff - Okay, I've:
  - removed routing;
  - removed extra module;
  - added missing CSS & scripts;
  - imported `MdToolbarModule`

… and called out—@jelbourn—the maintainer for review.

EDIT: Just updated to Node.JS 8.0.0 with npm 5.0.0; so added `package-lock.json`. Also some minor fixes that the previous commit left/broke.

SamuelMarks committed:
Added `@angular/router` back in; removed app.routes.ts; committed package-lock.json

jelbourn commented:
I hadn't really intended this app to live forever; IMO it's been superseded by https://github.com/angular/material.angular.io at this point. If you want to fork the repo, though, to serve as your own example you're totally welcome to it. 

I'll update the README to reflect the current status, and probably stop linking here from the material2 repo

SamuelMarks committed:
Updating
